src/Extensions/AzureBlobStorage/Program.cs

```diff
@@ -2,38 +2,43 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using Microsoft.Diagnostics.Tools.Monitor.Egress;
-using Microsoft.Diagnostics.Tools.Monitor.Egress.AzureBlob;
+using Microsoft.Diagnostics.Monitoring.AzureStorage.AzureBlob;
 using System.CommandLine;
-using System.CommandLine.Binding;
 using System.Text.Json;
 
 namespace Microsoft.Diagnostics.Monitoring.AzureStorage
 {
     internal class Program
     {
+        private readonly static string AzureBlobStorage = "AzureBlobStorage";
         private static Stream StdInStream = null;
         private static CancellationTokenSource CancelSource = new CancellationTokenSource();
+
+        protected static ILogger Logger { get; set; }
+
         static async Task<int> Main(string[] args)
         {
-            // Expected command line format is: AzureBlobStorage.exe Egress --Provider-Name MyProviderEndpointName
-            RootCommand rootCommand = new RootCommand("Egresses an artifact to Azure Storage.");
+            using var loggerFactory = LoggerFactory.Create(builder =>
```

Comment:
This should be done in the Egress method and not stored as static state on the Program class.